### ISSUE TITLE
Add information about the network management use-case

### ISSUE THREAD

**Issue by korvoj** [nmaas-platform/nmaas-docs#74]
On docs.nmaas.eu there is already a dedicated page for the virtual lab use-case. Similar page should be written for the original, network management use-case as well.

@llopat , do you remember if we already have some text that can be reused? Maybe this would be a good candidate - https://network.geant.org/nmaas/, but not sure if we would be allowed to mirror the content.

**Comment by llopat** [comment]
@korvoj yes, this text was prepared by me/Frederic so it can be re-used for sure as least as a starting point.
